### PR TITLE
DOCKER-450 [Add] Support for Alfresco v23.1 && v23.2

### PR DESCRIPTION
**Pull request by codingBenVdS**
DOCKER-450 [Add] Support for Alfresco v23.1 && v23.2
  * [Add] tomcat-embedded V10
  * [TODO] Add postgres V15 ?
  * [Sonarqube] ignored TomcatFactory.java from sonarqube. Duplicate code and 0% test issues due to code being seperated from tomcat-base source-code. (can be removed after Alfresco V7.X is dropped)
Ticket: https://xenitsupport.jira.com/browse/DOCKER-450

### CODE REVIEW

File: 2repository/23.1/community-23.1.0/overload.gradle [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+                    minor: 1,
+                    rev  : 0,
+                    label: 1
+            ],
+            flavor : 'community'
+    ]
+    solrimage = 'docker.io/xenit/alfresco-solr6:2.0.8.2'
+    shareimage = 'docker.io/xenit/alfresco-share-community:23.1.0'
+//    13,14,15 are supported
+    psqlimage = 'docker.io/xenit/postgres:13'
```
[the remark this answers is not in the document]
Comment (thijslemmens) [reply 2repository/23.1/community-23.1.0/overload.gradle:14]: Yes, or the latest. We don't maintain this image anymore. Better move to the official image.

**Review by thijslemmens — CHANGES_REQUESTED** [review]
Use the official PostgreSQL images. Remove the Jackson dependency now it is bundled in our logging library

File: 2repository/23.1/community-23.1.0/overload.gradle (hunk 1 above)
[the remark this answers is not in the document]
Comment (codingBenVdS) [reply 2repository/23.1/community-23.1.0/overload.gradle]: Using now the postgres:15 image

File: tomcat-base/build.gradle [hunk 2]
```diff
@@ -62,6 +70,7 @@ subprojects {
         main {
             java {
                 srcDir file("$project.parent.projectDir/src/shared/main/java")
+                srcDir file("$project.projectDir/src/shared/main/java")
```
Comment (thijslemmens) [inline tomcat-base/build.gradle:73]: Why did you add this? Could have just put it in main instead of shared/main?

**Review by thijslemmens — CHANGES_REQUESTED** [review]
Not sure why you need 3 versions of TomcatFactory now, we only used one in the passed. You should at most have 2?

**Review by thijslemmens — APPROVED** [review]

**Pull request merged**